packages/matchbox/src/components/Modal/Modal.module.scss

```diff
@@ -52,3 +52,10 @@
 .exited {
   pointer-events: none;
 }
+
+.CloseButton {
+  color: color(grey, 10);
```

Comment:
Not sure you want to use <Button> here, you'll need to override hover styles too right?
I'm okay with just an UnstyledLink

Reply:
The only hover style that was a problem was the `position: absolute` which I forces with `!important` :/.